README.md

```diff
@@ -12,9 +12,9 @@ This package makes it easy to store and retrieve some values. Stores values are
 It can be used like this:
 
 ```php
-$valuestore = Valuestore::make($pathToFile);
+$valuestore = Valuestore::make($pathToFile, ['key' => 'value']); // Create Valuestore and store values in a single step with the optional second argument
```

Comment:
Revert the changes, I think the new functionality is not important enough to warrant  a demo so high up in the readme.